src/api/app/models/workflow/step/configure_repositories.rb

```diff
@@ -8,7 +8,7 @@ class Workflow::Step::ConfigureRepositories < Workflow::Step
   validate :validate_architectures
 
   def call
-    return if workflow_run.closed_merged_pull_request? || workflow_run.reopened_pull_request?
+    return if workflow_run.closed_merged_pull_request? || workflow_run.reopened_pull_request? || workflow_run.unlabeled_pull_request?
```

Comment:
Why skip this step for labeling?

Reply:
This is only skipped for `unlabeled` action. There is nothing more to do in this step for `unlabeled` action. When the event is `labeled` we do execute this whole method.